src/Files.App/Views/Layouts/BaseGroupableLayoutPage.cs

```diff
@@ -321,6 +321,14 @@ protected async void RenameTextBox_KeyDown(object sender, KeyRoutedEventArgs e)
 				case VirtualKey.Right:
 					e.Handled = (textBox.SelectionStart + textBox.SelectionLength) == textBox.Text.Length;
 					break;
+				case VirtualKey.Home:
+					textBox.SelectionStart = 0;
```

Comment:
```suggestion
		if (!isShiftPressed)
					textBox.SelectionStart = 0;
```

Reply:
@hishitetsu do you agree with this suggestion? (see #13847 for reference)

Reply:
Yes, but indentation should be changed.